src/buildConfig.js

```diff
@@ -119,7 +123,7 @@ function addIgnores(config, configDir, ignorePath) {
   // Absoluteize config.ignoreFiles
   if (config.ignoreFiles) {
     config.ignoreFiles = [].concat(config.ignoreFiles).map(glob => {
-      if (path.isAbsolute(glob)) return glob
+      if (path.isAbsolute(glob.replace(/^\!/, ""))) return glob
```

Comment:
What is the point of this change? It doesn't seem related to anything else in this PR...